<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>210</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>210</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>210</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>350</width>
     <height>210</height>
    </size>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>401</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QFrame" name="Wrapper">
       <property name="styleSheet">
        <string notr="true">border: 1px solid #ccc;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="Result_Label">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>25</y>
          <width>168</width>
          <height>184</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid #ccc; background-color: #fff;</string>
        </property>
        <property name="text">
         <string>NONE</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="Result_Title">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>168</width>
          <height>25</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(103, 155, 203, 246); color: #fff;</string>
        </property>
        <property name="text">
         <string>RESULT</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="Camera_Title">
        <property name="geometry">
         <rect>
          <x>168</x>
          <y>0</y>
          <width>182</width>
          <height>25</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(103, 155, 203, 246); color: #fff;</string>
        </property>
        <property name="text">
         <string>CAMERA</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="Camera_Frame">
        <property name="geometry">
         <rect>
          <x>168</x>
          <y>24</y>
          <width>181</width>
          <height>186</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid #ccc; background-color: #fff;</string>
        </property>
        <property name="text">
         <string>CAMERA ERROR</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="Update_Button">
        <property name="geometry">
         <rect>
          <x>48</x>
          <y>184</y>
          <width>77</width>
          <height>23</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #fff; border: 1px solid #ccc;</string>
        </property>
        <property name="text">
         <string>UPDATE</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
